app/helpers/schemas/paginationSchema.js

```diff
@@ -0,0 +1,16 @@
+module.exports.paginationSchema = {
+  type: 'object',
+  properties: {
+    limit: {
+      type:'string',
```

Comment:
shouldn't this be a number type?

Reply:
Yes, but the query params are strings. When AJV tries to validate it returns an error.

> with type: 'number' 

![image](https://user-images.githubusercontent.com/31294096/141012779-1f84b6d3-0d4e-4d51-9759-0c268e9d1d97.png)

Reply:
Got it. Can you check if it's a numeric string? the same with the page parameter

Reply:
Ah, I just saw that you did just that with a regex, is there a way to do it with an AJV option? just for knowledge haha

Reply:
Yes, another solution is to enable Type Coercion when creating the AJV instance.
`const ajv = new Ajv({coerceTypes: true})`
in that case, you can use `type: number` in schema validator